modules/alloy/src/main/ts/ephox/alloy/positioning/layout/Placement.ts

```diff
@@ -0,0 +1,36 @@
+import { Optional } from '@ephox/katamari';
+import { Attribute, SugarElement } from '@ephox/sugar';
+
+export type Placement = 'north' | 'northeast' | 'northwest' | 'south' | 'southeast' | 'southwest' | 'east' | 'west';
```

Comment:
That really doesn't make sense honestly, as then you'd need to do it in every place we did that. It should just be a norm that we apply to everything.

That said, I think this actually might be able to be a `const enum` so will look at switching.

Reply:
For reference, the issues with an enum are that it can't treeshake. You can use a `const enum` to counteract that, however those can't be exported via a public API (as it's inlined when it's compiled). Neither case applies here, which is why I believe it can be a `const enum`.

Reply:
Have converted it to a `const enum`